test/lib/linter/rules/snapshots/NoDeprecatedApi.ts.md

```diff
@@ -54,15 +54,130 @@ Generated by [AVA](https://avajs.dev).
             severity: 2,
           },
           {
-            column: 24,
+            column: 3,
             line: 16,
+            message: 'Bootstrap parameter \'data-sap-ui-animation\' should be replaced with data-sap-ui-animation-mode',
+            messageDetails: 'Migrate to \'data-sap-ui-animation-mode\' attribute AnimationMode (https://ui5.sap.com/1.120/#/api/module:sap/ui/core/AnimationMode)',
+            ruleId: 'no-deprecated-api',
+            severity: 2,
+          },
+          {
+            column: 3,
+            line: 17,
+            message: 'Abandoned bootstrap parameter \'data-sap-ui-areas\' should be removed',
+            ruleId: 'no-deprecated-api',
+            severity: 1,
+          },
+          {
+            column: 3,
+            line: 18,
+            message: 'Redundant bootstrap parameter \'data-sap-ui-auto-aria-body-role\' should be removed',
+            ruleId: 'no-deprecated-api',
+            severity: 2,
+          },
+          {
+            column: 3,
+            line: 19,
+            message: 'Redundant bootstrap parameter \'data-sap-ui-binding-syntax\' should be removed',
+            ruleId: 'no-deprecated-api',
+            severity: 2,
+          },
+          {
+            column: 3,
+            line: 20,
+            message: 'Duplicate bootstrap parameter \'data-sap-ui-binding-syntax\' with value \'data-sap-ui-binding-syntax\'',
+            ruleId: 'no-deprecated-api',
+            severity: 1,
+          },
+          {
+            column: 3,
+            line: 20,
+            message: 'Redundant bootstrap parameter \'data-sap-ui-binding-syntax\' should be removed',
+            ruleId: 'no-deprecated-api',
+            severity: 1,
+          },
+          {
+            column: 3,
+            line: 22,
+            message: 'Redundant bootstrap parameter \'data-sap-ui-manifest-first\' should be removed',
+            messageDetails: 'Set the manifest parameter in component factory call',
+            ruleId: 'no-deprecated-api',
+            severity: 2,
+          },
+          {
+            column: 3,
+            line: 23,
+            message: 'Redundant bootstrap parameter \'data-sap-ui-no-duplicate-ids\' should be removed',
```

Comment:
I'm not sure whether redundant is the correct wording for this (and maybe also other parameters).
Allowing duplicate IDs is a feature that has been discontinued. But removing the parameter might break the application in UI5 1.x as it might use duplicate IDs in case the parameter was set to "false" (default "true").

Reply:
Maybe, the abandoned is the correct term here. Users should decide on their own and based on the details how to deal with such cases